tests/drivers/gpio/egpio_more_loops/testcase.yaml

```diff
@@ -0,0 +1,23 @@
+common:
+  sysbuild: true
+  tags:
+    - drivers
+    - gpio
+    - ci_tests_drivers_egpio
+  depends_on: gpio
+  filter: not CONFIG_COVERAGE
+  min_flash: 64
+  harness: ztest
+  harness_config:
+    fixture: five_gpio_loopbacks
+  platform_allow:
+    - nrf54l15dk/nrf54l15/cpuapp
+  integration_platforms:
+    - nrf54l15dk/nrf54l15/cpuapp
+
+tests:
+  drivers.egpio.many_loops_icmsg:
+    extra_args: SB_CONFIG_EGPIO_BACKEND_ICMSG=y
```

Comment:
There is also icbmsg backend. Will that not be tested?

Reply:
I see all tests are failing with ICBMSG backend.

Reply:
In basic_egpio_test I add one test case - a "stress" test.
Yesterday, I got test FAILED because when loop ends, rc was huge negative number like -2222546.
Thus, I added in the loop TC_PRINT() when rc is non zero. And now, I get timeout.
https://jenkins-ncs.nordicsemi.no/job/latest/job/sub/job/test-low-level/job/main/3236/testReport/junit/nrf54l15dk_nrf54l15_cpuapp_egpio_basic_api_drivers.egpio/loopback_icbmsg/Build_and_test___Moonlight_App_core___Moonlight_App_core___Build_and_run_Zephyr_tests___egpio_basic_api_drivers_egpio_loopback_icbmsg/

Reply:
ICBMSG was added